static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -168,9 +166,9 @@ function MemoryChart({
     },
     {
       seriesName: t('Free Heap Memory'),
-      data: memorySpans.map(span => ({
-        value: span.data.memory.totalJSHeapSize - span.data.memory.usedJSHeapSize,
-        name: span.endTimestamp,
```

Comment:
this was probably a timestamp in seconds

but whatever, now all the timestamp values inside this component are miliseconds, and all the places that read datetimes are updated to remove the `* 1000` stuff.